pkg/login/social/connectors/org_role_mapper.go

```diff
@@ -204,14 +205,26 @@ func (m *OrgRoleMapper) getAllOrgs() (map[int64]bool, error) {
 }
 
 func splitOrgMapping(mapping string) []string {
-	splitted := strings.Split(mapping, ":")
-	if len(splitted) <= 3 {
-		return splitted
+	result := make([]string, 0, 3)
+	re := regexp.MustCompile(":")
```

Comment:
Probably cheap to compile but we can move this to a package variable to we don't have to to it on each function call